validator/lib/validateTileset.js

```diff
@@ -32,6 +32,19 @@ function validateNode(root, parent, resolve) {
         var tile = node.node;
         var parent = node.parent;
 
+        if (defined(tile.content.boundingVolume)) {
+            var region = tile.content.boundingVolume.region;
+            var parentRegion = parent.content.boundingVolume.region;
```

Comment:
For this PR we aren't checking against the parent yet, instead check against `tile.boundingVolume`.